pyonfx/convert.py

```diff
@@ -44,12 +48,12 @@ class ColorModel(Enum):
 
 class Convert:
     """
-    This class is a collection of static methods that will help
+    This class is a collection of methods that will help
     the user to convert everything needed to the ASS format.
     """
 
-    @staticmethod
-    def time(ass_ms: Union[int, str]) -> Union[str, int, ValueError]:
+    @classmethod
```

Comment:
Since they use internal methods, I thought it made more sense. It doesn't really matter after all in the current state of `Convert` class. I can't revert this changes if you really want static methods.

Reply:
It makes more sense for them to be static, to me.

Reply:
Like I said, it interacts with internal methods but ok